front/app/components/admin/NavigationTabs/Tab.tsx

```diff
@@ -51,12 +56,32 @@ const Tab = styled.div`
     `border-color: ${darkSkyBlue};
     // border-color: ${colors.primary}; TODO : set accent color in component library
 
-    > * {
+    a {
         color: ${colors.primary};
     }`}
   `}
 `;
 
-type TabProps = { active: boolean } & React.HTMLAttributes<HTMLDivElement>;
+const StatusLabelWithMargin = styled(StatusLabel)`
```

Comment:
Minor: In these cases, we can put the StatusLabel in a Box and pass the margin on the Box to avoid using styled components

Reply:
I'll need to release the component library in this case. Do you think it's worth the effort?

Reply:
You don't need to change anything in the component library

I mean something like this

```
<Box ml="12px"><StatusLabel .......
```

Reply:
Ah good idea. Done here and in `front/app/components/admin/TabbedResource/Tab.tsx`